packages/@orchidui-vue/src/Builder/Form/OcFormBuilder.vue

```diff
@@ -17,6 +17,10 @@ import {
   SelectOptions,
   // TODO : add all form here , best load as async component
 } from "@/orchidui";
+
+import { TextEditor } from "@/orchidui/TextEditor";
```

Comment:
hi @kutsaniuk , i am sorry for this issue https://github.com/hit-pay/orchid/issues/75, please  register in local page first, later will add to form builder, and use slot #formName